### ISSUE TITLE
‘Raiding’ illager mobs don’t respond to Mob Enrager or Mob Army staffs

### ISSUE THREAD

**Issue by aarter180** [Flemmli97/MobBattle#9]
#### Issue (The better the description the better i can solve the issue):
Illagers in a vanilla village raid don’t respond to the Mob Enrager or Mob Army staffs and usually ignore the other mob attacking them. Sometimes a few of the raiding illagers do stop and fight back later after getting hit by their enemy, but it is unclear why it’s only sometimes.   

###Steps to reproduce:

Use the Mob Enrager or Mob Army staffs on illagers during a raid and another mob that can attack them.  At least most of the illagers will continue to patrol the village and ignore their attacker(s).

#### Affected Versions (Latest is NOT a version):

- Minecraft: 1.14.4
- Forge: 1.14.4-28.1.116
- MobBattle: 1.14.4-2.3.4

#### Other Mods (If you think it might be cause of an incompability):

- Mutant Beasts: 1.14.4-0.8.1
(I don’t think that this is a compatibility issue because I tested with only vanilla mobs but im including it just in case)

**Comment by Flemmli97** [comment]
i cant reproduce it. i created a raid and used both the enrager and army staff and both time they properly went after the correct target

**Comment by aarter180** [comment]
Sorry for the late response, I didn't seem to see a notification for this.  I'll try this again soon with what you said in mind.

**Comment by aarter180** [comment]
I tested it this time with no other mods and I can't reproduce the issue again.  Everything you said happened to you applied to me as well.  I'm not sure if other mods somehow made a difference before.

**Comment by aarter180** [comment]
I tested it once more along with the mutant beasts mod version listed in the original posts and now I had no issues.  They were fighting consistently save for an exception.  There was only a brief moment where raiders stopped fighting the enemy team I assigned and it was when the raiders all at once changed direction to pick up an Ominous banner the captain dropped.  I think it was easier for me to notice because I used an arena to tag all teammates and it was enclosed inside an arena instead of outside in a random world like I usually did.  So no chance for the banner to get lost or blown up.  That's my only lead for how I previously had this issue.  But I think that is a separate issue, the raiding illager mobs did respond consistently to the staffs.  I'm new to using Github for this so should I close this?
